packages/node-cli/.gitignore

```diff
@@ -1,2 +1,2 @@
 # config
-**/core/sdksetup/config*.json
\ No newline at end of file
+**/core/sdksetup/config.json
```

Comment:
Was there any reason for changing this from config*.json to config.json?

Reply:
No, I'll change it back. I was doing some testing and forgot to change it back.